AlimentacionSaludable/src/dominio/Ingesta.java

```diff
@@ -59,4 +59,12 @@ public boolean equals(Object obj) {
         return sonIguales;
     }
 
+    @Override
+    public int hashCode() {
+        int hash = 7;
+        hash = 83 * hash + Objects.hashCode(this.fechaDeIngesta);
+        hash = 83 * hash + Objects.hashCode(this.listaAlimentosPorFecha);
+        return hash;
+    }
+    
```

Comment:
Why you use differents hash functions for different classes?

Reply:
Cada clase tiene q tener su funcion HashCode y Netbeans te la autogenera con los atributos q elegis. HashCode necesita 2 numeros primos cualquiera, y estos son generados random por netbeans tmb.